willsCheatMenu/src/hacks/utility.ts

```diff
@@ -86,15 +87,19 @@ new Hack(category.utility, "Edit walkspeed").setClick(async () => {
 	await Toast.fire("Success!", `Successfully made walk speed ${parseFloat(walkSpeed.value) || 1.5}!`, "success");
 });
 
+let teleportingInterval = -1;
+
 new Toggler(category.utility, "Toggle Click Teleporting").setEnabled(async () => {
-	window.teleportingInterval = setInterval(() => {
+	teleportingInterval = setInterval(() => {
 		try {
 			_.player._playerContainer.walkSpeed = 500;
-		} catch (e) {}
+		} catch (e) { 
+			await Toast.fire("An error occurred.", `An error occurred while toggling click teleporting. Error: ${e}`, "error");
+		}
```

Comment:
this will spam toasts as fast as `setInterval` can speeeeed. :3

Reply:
This is a feature. It will emphasize the importance of the error.r